course_catalog/tasks.py

```diff
@@ -34,3 +40,48 @@ def get_edx_data():
             break
 
         url = response.json()["next"]
+
+
+@task
+def get_ocw_data():
+    """
+    Task to sync OCW course data with database
+    """
+    raw_data_bucket = boto3.resource(
+        's3',
+        aws_access_key_id=settings.OCW_CONTENT_ACCESS_KEY,
+        aws_secret_access_key=settings.OCW_CONTENT_SECRET_ACCESS_KEY
+    ).Bucket(name=settings.OCW_CONTENT_BUCKET_NAME)
+
+    # Get all the courses keys
+    ocw_courses = set()
+    # print("Assembling list of courses...")
+    for file in raw_data_bucket.objects.all():
+        key_pieces = file.key.split("/")
+        course_prefix = key_pieces[0] + "/" + key_pieces[1]
+        if course_prefix not in NON_COURSE_DIRECTORIES:
+            if "/".join(key_pieces[:-2]) != "":
+                ocw_courses.add("/".join(key_pieces[:-2]))
+
+    # Query S3 Bucket for JSONs per course
+    for course_prefix in ocw_courses:
+        loaded_raw_jsons_for_course = []
+        last_modified_dates = []
+        # print("Digesting: " + course_prefix + " ...")
+        for obj in raw_data_bucket.objects.filter(Prefix=course_prefix):
+            loaded_raw_jsons_for_course.append(load_json_from_string(obj.get()["Body"].read()))
```

Comment:
The idea of a helper function was to handle corrupted JSON files (there is at least one that I am aware of).  `load_json_from_string` doesn't handle that yet, but it will eventually.